microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -94,9 +98,16 @@ def __execute_assert_zero(
         self, cmd: str, path: PurePath, timeout: int = 600
     ) -> str:
         result = self.node.execute(
-            cmd, sudo=True, shell=True, cwd=path, timeout=timeout
+            cmd,
+            sudo=True,
+            shell=True,
+            cwd=path,
+            timeout=timeout,
+            expected_exit_code=0,
+            expected_exit_code_failure_message=(
+                f"{cmd} failed with nonzero error code"
```

Comment:
The command will be printed in the message, so it doesn't need to be printed again.

Reply:
I was going to get rid of this function in my next commit to give more specific error messages on all these build commands. Can we leave this and I'll get rid of it in the next PR?

Reply:
(changed this one)